### PR TITLE
Use different rgbd::Image constructor

### PR DESCRIPTION
**Pull request by MatthijsBurgh**
Merge together with tue-robotics/rgbd#26

### CODE REVIEW

File: src/Kinect.cpp [hunk 1]
```diff
@@ -22,9 +23,21 @@ float randomUniform(float min, float max)
...
-    cam_model_.setFocalLengths(554.2559327880068, 554.2559327880068);
-    cam_model_.setOpticalCenter(320.5, 240.5);
-    cam_model_.setOpticalTranslation(0, 0);
+    sensor_msgs::CameraInfo cam_info;
+    cam_info.K = sensor_msgs::CameraInfo::_K_type({554.2559327880068, 0.0, 320.5,
+                          0.0, 554.2559327880068, 240.5,
+                          0.0, 0.0, 1.0});
+    cam_info.P = sensor_msgs::CameraInfo::_P_type({554.2559327880068, 0.0, 320.5, 0.0,
+                                                   0.0, 554.2559327880068, 240.5, 0.0,
+                                                   0.0, 0.0, 1.0, 0.0});
```
Comment (jlunenburg) [inline src/Kinect.cpp:32]: Basically, you put the same numbers in a `PinholeCameraModel` and extract focal length, optical center and optical translation from there. 
* Why the change?
* Is the result any different?
* Is it still necessary to keep `cam_model_` as a member?

**Review by jlunenburg — COMMENTED** [review]
As might be clear from my comment and corresponding questions, I'm not an expert on these models. One general comment: there are no tests for any of this code, did you test this manually?

File: src/Kinect.cpp (hunk 1 above)
Comment (MatthijsBurgh) [reply src/Kinect.cpp:32]: The `depth_camera` is removed from Image, as it isn't used.

No the result isn't different.

Because I changed the same of the `depth_camera` from `cam_model_` to `rasterizer_`. But `cam_model_` it still used at the end of the file.

**Comment by MatthijsBurgh** [comment]
Jep `amigo-start` still works and the `amigo-show-kinect` still show normal images.

File: src/Kinect.cpp (hunk 1 above)
Comment (jlunenburg) [reply src/Kinect.cpp:32]: Thanks, based on your answer I had a look at https://github.com/tue-robotics/rgbd/pull/26 and now I see your point. Seems to make sense to me.

Seems a lot of code for this though but then again, I guess that these parameters might be Kinect-specific.

**Review by jlunenburg — APPROVED** [review]
As long as you're sure it still works I approve.

N.B.: if you want to merge this now it might be wise to also test this with the master branch of rgbd (in case you haven't already). Otherwise, you might also postpone merging until you're sure there won't be any further API changes in rgbd (e.g., file renaming)

File: src/Kinect.cpp (hunk 1 above)
Comment (MatthijsBurgh) [reply src/Kinect.cpp:32]: The matrices have a special structure which I filled in with the values which were already there.

**Comment by MatthijsBurgh** [comment]
This needs to be merged together with tue-robotics/rgbd#26
